Makefile

```diff
@@ -90,3 +92,9 @@ install: setup.data rbac_static.csv
 	scripts/install.sh 755 block_device_io.native $(DESTDIR)$(LIBEXECDIR)/block_device_io
 # Libraries
 	ocaml setup.ml -install
+# xenserver html docs
+	mkdir -p $(DESTDIR)$(DOCDIR)/html/xenserver
+	cp -r -L _build/ocaml/doc/api $(DESTDIR)$(DOCDIR)/html/xenserver
+	cp ocaml/doc/{*.js,*.html,*.css,*.png} $(DESTDIR)$(DOCDIR)/html/xenserver
```

Comment:
No. Each line in a Make recipe is executed in its own shell - it does not matter in what PWD a process is at the end because the shell will terminate.